Cargo.toml

```diff
@@ -29,7 +29,7 @@ gdk3x11 = {package = "gdkx11", version = "0.16", optional = true}
 gdk3wayland = {package = "gdkwayland", version = "0.16", optional = true}
 gtk3 = {package = "gtk", version = "0.16", optional = true}
 
-gdk4wayland = {package = "gdk4-wayland", version = "0.5", optional = true}
+gdk4wayland = {package = "gdk4-wayland", version = "0.5", optional = true, features = ["wayland_crate"]}
```

Comment:
I don't think this is correct, the feature should be enabled only if `gtk4_wayland` is enabled.

Reply:
not sure what you mean here?

Reply:
that looks correct to me